symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkDatafeedConfig.java

```diff
@@ -11,17 +11,26 @@
 @API(status = API.Status.STABLE)
 public class BdkDatafeedConfig {
 
-    private String version = "v1";
-    private String idFilePath;
-    private BdkRetryConfig retry = new BdkRetryConfig(BdkRetryConfig.INFINITE_MAX_ATTEMPTS);
+  private String version = "v1";
+  private String idFilePath;
+  private BdkRetryConfig retry = new BdkRetryConfig(BdkRetryConfig.INFINITE_MAX_ATTEMPTS);
+  private Boolean reuseDatafeedId;
 
-    public String getIdFilePath() {
-        if (idFilePath == null || idFilePath.isEmpty()) {
-            return "." + File.separator;
-        }
-        if (!idFilePath.endsWith(File.separator)) {
-            return idFilePath + File.separator;
-        }
-        return idFilePath;
+  public String getIdFilePath() {
+    if (idFilePath == null || idFilePath.isEmpty()) {
+      return "." + File.separator;
     }
+    if (!idFilePath.endsWith(File.separator)) {
+      return idFilePath + File.separator;
+    }
+    return idFilePath;
+  }
+
+  public boolean getReuseDatafeedId() {
```

Comment:
I'd perhaps rename it to `isReuseDatafeedId()`, WDYT?

Reply:
Sure.